build_library/catalyst_toolchains.sh

```diff
@@ -16,10 +16,13 @@ configure_target_root() {
     local cross_chost=$(get_board_chost "$1")
     local profile=$(get_board_profile "${board}")
 
+    # Get the host CBUILD.
+    local cbuild="$(portageq envvar CBUILD)"
+
     CHOST="${cross_chost}" \
         ROOT="/build/${board}" \
         SYSROOT="/usr/${cross_chost}" \
-        _configure_sysroot "${profile}"
+        _configure_sysroot ${cbuild} "${profile}"
```

Comment:
Feel free to comment on the ordering considering I got it wrong, I had misunderstood the order in which the shell sets the variables and evaluates `"$(portageq envvar CBUILD)"` in my example.